docs/tutorials/extending_admin_ui/5_creating_a_udw_tab.md

```diff
@@ -0,0 +1,536 @@
+# Creating a UDW tab
+
+The Universal Discovery Widget (UDW) is a separate React module.
+By default it contains two tabs: Browse and Search.
+
+![Universal Discovery Widget](img/udw_before.png)
+
+In this step you will add a new tab called Images which will display all Content items of the type 'Image'
+using a gallery of thumbnails.
+
+!!! tip
+
+    To be able to view the results of this step, create a few Content items of the type "Image".
+
+## Add a tab
+
+First add a `src/EzSystems/ExtendingTutorialBundle/Resources/public/js/add.tab.js` which governs the creation of the new tab:
+
+``` js hl_lines="11"
+(function (global, eZ) {
+    eZ = eZ || {};
+    eZ.adminUiConfig = eZ.adminUiConfig || {};
+    eZ.adminUiConfig.universalDiscoveryWidget = eZ.adminUiConfig.universalDiscoveryWidget || {};
+    eZ.adminUiConfig.universalDiscoveryWidget.extraTabs = eZ.adminUiConfig.universalDiscoveryWidget.extraTabs || [];
+
+    eZ.adminUiConfig.universalDiscoveryWidget.extraTabs.push({
+        id: 'images',
+        title: 'Images',
+        iconIdentifier: 'image',
+        panel: eZ.modules.ImagesPanel, // React component that represents content of a tab
+        attrs: {}
+    });
+
+    console.log('add.tab', eZ.adminUiConfig.universalDiscoveryWidget.extraTabs);
+})(window, window.eZ);
+```
+
+The highlighted line indicates the actual panel inside the tab that will display the images.
+
+## Provide uncompiled files
```

Comment:
These files I would not call `uncompiled files`. They are rather ReactJS development files.